issue-triaging.md

```diff
@@ -44,6 +44,72 @@ between the involved parties to allow work to begin or for the issue to be close
+   - For new components requests:
+      - Does the user explain the purpose and intended use of the component?
+      - Has the user provided example configurations, interfaces, or scenarios where the component would be applied?
+   - If the issues lacks sufficient information, ask the author clarifying questions:
+      - Collector configuration (missing or incomplete configurations are not uncommon)
+      - The expected vs. actual behavior
+      - Clear and complete steps to reproduce the issue
+      - Collector version
+      - Any other questions that might fit
+
+2. If the issue is deemed valid and requires attention, apply appropriate labels based on its nature:
+   - For bugs, add the `bug` label along with `waiting-for-codeowners`.
+   - For enhancements, use the `enhancement` label and also add `waiting-for-codeowners`.
+   - If the issue is about a new component, add the `sponsor-needed` label. We should encourage the user to join our SIG meeting and propose the new component to a broader audience. They can also reach out in the #otel-collector-dev channel on CNCF Slack.
+   - Check out [state diagram](#state-diagram) for detailed information about all labels.
+
+3. An issue may be considered invalid for the following reasons:
+   - The user is using an incorrect configuration.
+       - In such cases, a brief comment and a link to documentation should be sufficient to close the issue.
+   - The issue was filed in the wrong repository:
+       - For example, we might receive an issue for `otlpexporter` in contrib. It is best to reach out to project maintainers on CNCF slack and ask to transfer the issue.
+
+4. An issue may be considered complete for following reasons:
+   - The enhancement/bugfix PR has been merged.
+   - The enhancement/bugfix has already been addressed in the newer version.
+   - The new component has been added to the repository.
+
+5. _For new components_:
+   - We should set clear expectations reagarding a new component, if it is accepted:
+      - Will the user contribute the necessary code, or they are expecting someone from upstream to volunteer?
+      - Will the user be available to provide feedback, clarify requirements, or help with testing and maintenance?
+
+
+#### Triage process flowchart 
+
+```mermaid
+flowchart TD
+    Start{Check validity}
+
+    Start -->|Unclear| WaitingAuthor[Tag:<br><code>waiting-for-author</code><br>Ask for config, repro steps,<br>expected vs actual result]
+    Start -->|Invalid| Transfer[Close or<br>transfer issue:<br>wrong repo, bad config, etc.]
+    Start -->|Valid| Classify
+
+    Classify -->|Bug| EnrichBug[Tag:<br><code>bug</code>,<br><code>waiting-for-codeowners</code>,<br>Optional: <code>bug:perf/crash</code>,<br><code>workaround:yes/no</code>,<br><code>release:blocker</code>]
+    EnrichBug --> WaitingCodeowners[Tag:<br><code>waiting-for-codeowners</code>]
```

Comment:
What's the value of having a whole box dedicated to `Tag: waiting-for-codeowners`, when each of the boxes pointing to it also say to apply the `waiting-for-codeowners` label? I'd suggest we either remove the label from the separate boxes and have a single step for apply the tag in each case, or have each case apply the tag and remove the extra individual step.